http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -261,11 +281,9 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
         case None => Vector.empty
         case Some(samVisibleRuntimes) =>
           val samVisibleRuntimesSet = samVisibleRuntimes.toSet
-          // Making the assumption that users will always be able to access runtimes that they create
-          // Fix for https://github.com/DataBiosphere/leonardo/issues/821
           runtimes
             .filter(c =>
-              c.auditInfo.creator == userInfo.userEmail || samVisibleRuntimesSet
```

Comment:
I see you've consistently removed this (that's great, this sucks, lol). I think I need clarity on what the behavior is meant to be, now, in the several different edge cases that might arise (app|runtime creator leaves workspace, workspace member sees another's app|runtime, app|runtime errors)

Reply:
- if a creator leaves a workspace, they will no longer be able to access any of that workspace's (project's) resources, which is the intended behavior behind this security patch
- the behavior for workspace/project readers and owners shouldn't change
- if they're a reader they should only be able to access resources they have read+ permission on in Sam
- if they're an owner they should be able to access everything in the workspace/project

I'm not sure what error cases you mean here, an errored app/runtime should have the same permissions as a running one I think? And a resource is only set as deleted in Sam after its transitioned to a 'Deleted' status in Leo

Reply:
Nice, I think your final clarification is what I needed. v1 resources just don't have the sync issues that WSM-managed ones do